scalding-beam/src/main/scala/com/twitter/scalding/beam_backend/BeamMode.scala

```diff
@@ -74,3 +78,54 @@ object BeamSink extends Serializable {
         pc.asInstanceOf[PCollection[String]].apply(TextIO.write().to(path))
     }
 }
+
+class BeamFileIO[T](output: String) extends BeamSink[T] {
+  override def write(
+      pipeline: Pipeline,
```

Comment:
you never use the `Pipeline` right? That is just because you are reusing the BeamSink interface? Does it need the pipeline?

Reply:
Refactored the interface. Pipeline can be accessed from PCollection via `pcol.getPipeline`